wait-for

```diff
@@ -94,15 +96,13 @@ wait_for() {
       exit 0
     fi
 
-    if [ "$TIMEOUT" -le 0 ]; then
```

Comment:
This breaks the `--timeout 0` behaviour, now the script immediately returns "Operation timed out"

Reply:
@eficode

Reply:
You're right, I will give it a look.

Reply:
actually I've now tested a couple of releases and it seems not to work in any version..
I would expect to "./wait-for --timeout 0 localhost:12345" to wait forever, but it does not.. or am I missing something?

Reply:
Hmm, the issue is that we haven't had a test to verify this behavior*. I'm first adding that.

*Testing whether something never times out is not going to be possible, but at the very least I add a test which ensures it doesn't immediately timeout.